ES/ESmy021.xml

```diff
@@ -103,82 +175,108 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                        <seg type="rubrication">Divine names; name of the Metropolitan; part of the
                            name of the owner; headings and incipits of the texts; directives for the
                            ministers; abbreviations; elements of the punctuation signs and Ethiopic
-                           numerals.</seg>
+                           numerals. Rubrication has been carefully executed in <ref target="#h1">Hand 1</ref>hand
+                           along with the copying of the text.</seg>
+                        <desc>Very irregular, by a mediocre scribe. Somewhat more regular starting from
+                          <locus target="#24ra"/>. Broad, slightly right-sloping script.
+                          The scribe modified the proportions
+                          of his handwriting (cp., e.g., <locus target="#24r"/>, or <locus target="#34r"></locus>),
+                          or wrote in smaller or compressed script, probably in order to accomodate
+                           more text on the page (cp. <locus from="1va" to="5vb"/>,
+                           <locus from="11ra" to="12rb"/>, <locus from="15r" to="19v"/>).
+                            The word divider is always
+                            written with two additional red dots above and below.
+                           Despite differences,
+                            the entire book was likely copied by
+                            only one scribe.</desc>
                      </handNote>
-                     <handNote xml:id="h2" script="Ethiopic">Handwriting varies greatly throughout
-                        the text; however, the Ms. was most probably executed by one and the same
-                        scribe.<!--The names of this hands need to be replaced by appropriate refs.--></handNote>
-                  </handDesc>
+
+                      </handDesc>
                   <decoDesc>
 
-                     <decoNote type="frame" xml:id="d1">- <locus target="#19r">19r</locus>: Two
-                        narrow coloured (red, green, black) ornamental bands over cols. a, b, at the
-                        incipit of Text II.</decoNote>
-                     <decoNote type="miniature" xml:id="d2"/>
+                     <decoNote type="band" xml:id="d1">
+                      <locus target="#19r"/>
+                      <desc>Two crude narrow coloured (red, green, black) ornamental bands
+                        over the text columns, at the incipit of <ref target="#ms_i1.5"/>.</desc>
+                      </decoNote>
                   </decoDesc>
                   <additions>
                      <list>
-                        <item xml:id="e1">- Text is written in (smaller letters and) compressed script: fols. 15r-v, 17r. - Crude musical notation signs on fols. 19r, 22r-23v and elsewhere. - Crudely drawn lines delimiting sections of the text: e.g., fols. 20rb, 29ra, 30ra.</item>
```

Comment:
is the content of other extras lost or moved somewhere?

Reply:
removed